sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalog.scala

```diff
@@ -322,9 +422,88 @@ class CosmosCatalog
     }
   }
 
-  @throws(classOf[UnsupportedOperationException])
-  override def renameTable(oldIdent: Identifier, newIdent: Identifier): Unit = {
-    throw new UnsupportedOperationException("renaming table not supported")
+  @tailrec
+  private def deleteViewTable(databaseName: String, viewName: String): Boolean = {
+    logInfo(s"deleteViewTable DB:$databaseName, View: $viewName")
+
+    this.viewRepository match {
+      case Some(viewRepositorySnapshot) =>
+        viewRepositorySnapshot.getLatest() match {
+          case Some(viewDefinitionsSnapshot) =>
+            val lastBatchId = viewDefinitionsSnapshot._1
+            val viewDefinitions = viewDefinitionsSnapshot._2
+
+            viewDefinitions.find(v => v.databaseName.equalsIgnoreCase(databaseName) &&
```

Comment:
cosmos db "ids" and database name is case sensitive.

any reason we are using `equalsIgnoreCase` on the databasename.

here and in all other cases here that we are doing `equalsIgnoreCase` on dbName or containeName

Reply:
No - it was a bug. Thanks for catching it.